tools/start_server.py

```diff
@@ -314,10 +342,10 @@ def install_app(self, _app_id, _package_url):
   def add_app_to_file_server(self, _app_storage_folder):
     secret_key = str(uuid.uuid4())
     print('\033[2mSecret key is %s\033[0m' % (secret_key))
-    WebEngineManager.file_server.add_app_mapping(secret_key, _app_storage_folder)
+    self.file_server.add_app_mapping(secret_key, _app_storage_folder)
     return {
       "key": secret_key, 
-      "url": 'http://%s:%s/%s/' % (WebEngineManager.file_server.REMOTE_HOST, WebEngineManager.file_server.REMOTE_PORT, secret_key)
+      "url": 'http://%s:%s/%s/' % (self.file_server.REMOTE_HOST, self.file_server.REMOTE_PORT, secret_key)
```

Comment:
It should effectively be the same thing at the moment. Since it's already part of the instantiated file_server object as well use the properties 🤷‍♂️

Reply:
Referring back to the multiple instances comment, having those properties is required for having multiple instances of the file server

Reply:
Yup. My fault! I didn't understand what you meant.